### ISSUE TITLE
Broken Link in Wiki

### ISSUE THREAD

**Issue by UntotaufUrlaub** [andyljones/reinforcement-learning-discord-wiki#1]
The Link in 
"Official OpenAI environments, and third party environments"
in the section "RL Environments" within the wiki does lead to 404 Error for me.
Is this Link still valid?

**Comment by ask4jubad** [comment]
> environments

Yes, you are right, the link is broken and it appears the file is no longer available even on the project repo.

The maintainers have moved to a new repo here:  [Gym -> Gymnasium](https://github.com/Farama-Foundation/Gymnasium)

Feel free to check it out.
